geopandas/tools/tests/test_sjoin.py

```diff
@@ -365,9 +364,7 @@ def test_right(self, predicate, dfs):
             )
             exp.index.names = df2.index.names
 
-        # GH 1364 fix of behaviour was done in pandas 1.1.0
-        if predicate == "within" and Version(pd.__version__) >= Version("1.1.0"):
-            exp = exp.sort_index()
+        exp = exp.sort_index()
```

Comment:
Shouldn't `if predicate == "within"` stay?

Reply:
yep, that's an error in refactoring, sorting only makes a difference for within, redundante for other predicates.